gemfiles/rails_6.0.gemfile

```diff
@@ -2,6 +2,6 @@
 
 source "https://rubygems.org"
 
-gem "rails", "~> 6.0"
+gem "rails", "6.0.6.1"
```

Comment:
yes. for some reason, leaving the gem 'rails' as '~> 6.0' will make the gemfile.lock to generate Rails 6.1.7.7 instead of 6.0.6.1. We wanted to test 6.0 here, so that's why I put more precise version in here. Also, this version of 6.0.6.1 is guaranteed to be the latest one